src/includes/class-alma-wc-payment-gateway.php

```diff
@@ -863,15 +863,14 @@ public function woocommerce_available_payment_gateways( $_available_gateways ) {
 
 		$new_available_gateways = array();
 		foreach ( $_available_gateways as $gateway ) {
+			$new_available_gateways[] = $gateway;
```

Comment:
I think you should use `$key` like this:

```
   foreach ( $_available_gateways as $key => $gateway ) {	 
       $new_available_gateways[$key] = $gateway;

```

because original `$_available_gateways` is formated like this =>

```
[
  'alma' => $the_alma_gateway_object,
  'paypal' => $the_paypal_gateway_object,
]
```

with your modification, we will have something like that:

```
[
  0 => $the_alma_gateway_object,
  'alma_pnx_plus_4' => $the_alma_gateway_object,
  'alma_pay_later' => $the_alma_gateway_object,
  1 => $the_paypal_gateway_object,
]

```

isn't it ?

Reply:
Yes it is.